src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -74,9 +98,19 @@ const SidebarView = View.extend({
   },
   className: 'sidebar flex-region',
   template: ClinicianSidebarTemplate,
+  templateContext() {
+    const showDelete = !this.model.isNew() && !this.model.get('last_active_at');
```

Comment:
I think this can also be `this.model.canEdit()` as below with no check to `isNew`  On all of the other sidebars you can delete a new sidebar model with the same method.  Maybe the method should be a name that indicated edit or delete?  `canModify`?

Reply:
_although_ really `!this.model.get('last_active_at')` or `{{#unless last_active_at}}` is about the same.

Reply:
I'm thinking of using the method on the model in all scenarios here. It's not quite as clean in some cases, but it will at least be consistent if the criteria for editing a clinician ever changes.